### DISCUSSION TITLE
Copilot stopped working entirely in VS2022 17.4 Preview 1 after subscription

### DISCUSSION THREAD

**Discussion by markw-t** [community/community#31502] [Copilot Conversations]
Posting here on behalf of Visual Studio customer who raised a feedback ticket 
[here](https://developercommunity.visualstudio.com/t/Copilot-does-not-work-with-VS/10130608)

[Deepak](https://developercommunity.visualstudio.com/user/19115/profile)
-
Reported Aug 24, 2022
[severity:It’s more difficult to complete my work]

After Installing
Microsoft Visual Studio Community 2022 (64-bit) - Preview
Version 17.4.0 Preview 1.0

Copilot Stopped Working

To clarify I have paid 10$ and completed activation etc, VSCode works fine.

Copilot was working before the Monthly Subscription began.
I have been using VS and Copilot for 2 moths with no issues

**Comment by BMustafa97** [comment]
same here

**Reply by tilmann1** [reply]
Can you please try again with the latest version of the VS extension? The issue should be fixed now. Thanks!

**Reply by chenhw** [reply]
> Can you please try again with the latest version of the VS extension? The issue should be fixed now. Thanks!

this issue remain. An error occurred for 'TriggerInlineSuggestionCommandArgs': System.OperationCanceledException uhm

**Comment by AbbottF** [comment]
I just got the following error with new VS 17.4 and Copilot upgrade. Thought this would be the place to post it ...
Abbott
![Copilot VS 17 4  Errors](https://user-images.githubusercontent.com/229561/201944235-9af2c9bd-cc59-4cff-af58-f38a6c52b17a.png)
[Copilot Error Output.txt](https://github.com/community/community/files/10013504/Copilot.Error.Output.txt)

**Reply by tilmann1** [reply]
The error message means that the Visual Studio Copilot extension was unable to launch the Copilot Agent helper process. Any chance you have a firewall in place that prevents unauthorized applications from using the network?

**Comment by Dmutro777** [comment]
Отправлено с устройства Galaxy
-------- Исходное сообщение --------От: Abbott Fleur ***@***.***> Дата: 15.11.2022  16:30  (GMT+02:00) Кому: community/community ***@***.***> Копия: Subscribed ***@***.***> Тема: Re: [community/community] Copilot stopped working entirely in VS2022 17.4 Preview 1 after subscription (Discussion #31502) 
I just got the following error with new VS 17.4 and Copilot upgrade. Thought this would be the place to post it ...
Abbott

Copilot Error Output.txt

**Comment by AbbottF** [comment]
Thanks. Aside from MS Defender firewall. I don’t see any settings there for GitHub or CoPilot. Strange. Seems MS would go out of its way to allow access.

 

??

Abbott

**Reply by jeffreyhunter77** [reply]
You will see the "The specified executable is not a valid application for this OS platform" error if your system's architecture is different from the binary. Right now it requires AMD 64 / x86 64, so if you're trying to use the Copilot extension on an ARM system or a 32 bit system, that's the error you'll get.

What does it show for System Type if you open the System Information app? It should show "x64-based PC":

<img width="748" alt="Screen Shot" src="https://user-images.githubusercontent.com/33921/202265101-bf85a665-5ae5-41b8-9b1c-4d150c7e0a3c.png">

**Comment by AbbottF** [comment]
It’s x64

**Reply by jeffreyhunter77** [reply]
I suppose another possibility would be if the file somehow became corrupted. A newer version (1.58.0.0) has been released since you originally posted. If you haven't already, I'd try upgrading the extension to the latest and see if you are still getting the error.
